examples/NIMF-Metal-2-AWS-connection/versions.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 1.5.4"
+  required_providers {
+    equinix = {
+      source  = "equinix/equinix"
+      version = ">= 1.20.0"
```

Comment:
We shouldn't add new content with old versions of the Terraform provider. This will need to be updated to be the latest version of the release after https://github.com/equinix/terraform-provider-equinix/pull/626/files#diff-703d45b76399eb49451408ebb9688abdad1236c9cfb95664c5044fe2488e19fb is merged into the provider.